question/classes/bank/creator_name_column.php

```diff
@@ -17,9 +17,11 @@
 /**
  * A column type for the name of the question creator.
  *
- * @package   core_question
- * @copyright 2009 Tim Hunt
- * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @package    core_question
+ * @copyright  2009 Tim Hunt
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @deprecated since Moodle 4.0
```

Comment:
deprecation info will go to the class document

Reply:
Moved depreciation in class documentation